[fix]Fixed most SDK deprecated links

apienx opened the pull request.

mtrunkat commented on docs/api/Apify.md:
```diff
@@ -60,7 +60,7 @@ effect.
 <tr>
 <td colspan="3"><p>Array of event types, which you can set for actor run, see
-  the <a href="https://apify.com/docs/webhooks#events-actor-run" target="_blank">actor run events</a> in the Apify doc.</p>
+  the <a href="https://docs.apify.com/webhooks#events-actor-run" target="_blank">actor run events</a> in the Apify doc.</p>
```
For example, this link is still wrong

`https://docs.apify.com/webhooks#events-actor-run`

should be

`https://docs.apify.com/webhooks/events#actor-run`

as the webhooks page was split into multiple once. Could you check out the links in this PR that contain hash part?

mtrunkat commented on docs/api/Apify.md:
````diff
@@ -484,7 +484,7 @@ Returns a new object which contains information parsed from all the `APIFY_XXX`
 ```
 
-For the list of the `APIFY_XXX` environment variables, see <a href="https://apify.com/docs/actor#run-env-vars" target="_blank">Actor
+For the list of the `APIFY_XXX` environment variables, see <a href="https://docs.apify.com/actor#run-env-vars" target="_blank">Actor
````
Also for example here, it should be `https://docs.apify.com/actor/run#environment-variables`

mnmkng commented on src/utils.js:
```diff
@@ -72,15 +72,15 @@ export const logSystemInfo = () => {
 /**
  * Gets the default instance of the `ApifyClient` class provided
- * <a href="https://apify.com/docs/sdk/apify-client-js/latest"
+ * <a href="https://sdk.apify.com/docs/api/apify#apifyclient"
```
This should point to `apify-client` docs, not back to self.

jancurn replied:
@apienx pls can you fix those, so that we can merge it?

jancurn replied:
Oh, these are already fixed

mnmkng commented on src/utils.js:
```diff
@@ -72,15 +72,15 @@ export const logSystemInfo = () => {
  * [`Apify.getValue()`](#module_Apify.getValue) or [`Apify.call()`](#module_Apify.call).
  * The settings of the client can be globally altered by calling the
- * <a href="https://apify.com/docs/sdk/apify-client-js/latest#ApifyClient-setOptions"
+ * <a href="https://sdk.apify.com/docs/api/apify#apifyclient#ApifyClient-setOptions"
```
Same as above.

apienx commented:
This one addresses the comments.

```
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/scraping=https://docs.apify.com/scraping=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/actor#run-env-vars=https://docs.apify.com/actor/run#environment-variables=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/actor#run-lifecycle=https://docs.apify.com/actor/run#lifecycle=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/actor#input-output=https://docs.apify.com/actor/run#input-and-output=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/actor#container-web-server=https://docs.apify.com/actor/run#container-web-server=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/actor#quick-start=https://docs.apify.com/actor/quick-start=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/actor#base-images=https://docs.apify.com/actor/build#base-images=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/actor#build=https://docs.apify.com/actor/build=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/actor#run=https://docs.apify.com/actor/run=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/actor=https://docs.apify.com/actor=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/tasks=https://docs.apify.com/tasks=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/storage#queue=https://docs.apify.com/storage/request-queue=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/storage#dataset=https://docs.apify.com/storage/dataset=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/storage#key-value-store=https://docs.apify.com/storage/key-value-store=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/storage#queue=https://docs.apify.com/storage/request-queue=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/storage#dataset=https://docs.apify.com/storage/dataset=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/storage#key-value-store=https://docs.apify.com/storage/key-value-store=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/storage=https://docs.apify.com/storage=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/proxy=https://docs.apify.com/proxy=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/webhooks#events-actor-run=https://docs.apify.com/webhooks/events#actor-run=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/webhooks#adhoc=https://docs.apify.com/webhooks/ad-hoc-webhooks=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/webhooks=https://docs.apify.com/webhooks=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/cli=https://docs.apify.com/cli=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/sdk/apify-client-js/latest=https://docs.apify.com/api/apify-client-js/latest=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/sdk/apify-runtime-js/latest=https://sdk.apify.com/=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/api/apify-client-js/latest#ApifyClient-requestQueues-getQueue=https://docs.apify.com/api/apify-client-js/latest#ApifyClient-requestQueues=g' {} \;
find ./ -type f -exec sed -i -e 's=https://apify.com/docs/api/apify-client-js/latest#ApifyClient-datasets-getDataset=https://docs.apify.com/api/apify-client-js/latest#ApifyClient-datasets-getDataset=g' {} \;
```
